packages/vscode-ui5-language-assistant/package.json

```diff
@@ -33,6 +33,21 @@
       "type": "object",
       "title": "UI5 Language Assistant",
       "properties": {
+        "UI5LanguageAssistant.logging.level": {
+          "type": "string",
+          "enum": [
+            "off",
+            "fatal",
+            "error",
+            "warn",
+            "info",
+            "debug",
+            "trace"
+          ],
+          "default": "error",
+          "description": "The verbosity of logging according to the following order: trace > debug > info > warn > error > fatal > off.",
+          "scope": "resource"
```

Comment:
The scope should be `"window"`. Resource-level scope means each workspace folder could have a different value, and when you access the configuration property you have to send a resource URI. The log level however is not in the scope of a resource/workspace folder, it's general for the entire extension (in the current running instance).

Reply:
I am pretty sure we made this mistake in all our extensions :(

Reply:
Yes, probably because this is what the [example](https://github.com/SAP/vscode-logging/blob/33b09d75f368d34ce5d5fc92199928c270a21786/examples/extension/package.json#L39) says. But I fixed it in hana-explorer at least.